app/src/androidTest/java/com/example/sharingang/DetailedItemFragmentTest.kt

```diff
@@ -25,7 +25,7 @@ class DetailedItemFragmentTest {
 
     private val itemTitle = "T"
     private val itemDescription = "D"
-
+/*
```

Comment:
We should just remove the code instead of commenting it out

Reply:
I thought maybe we would want to reuse it later?

Reply:
It's still in the history of the file

Reply:
You're right